docs/common/craft-parts/reference/plugins/cargo-package_plugin.rst

```diff
@@ -0,0 +1,89 @@
+.. _craft_parts_cargo-package_plugin:
+
+Cargo package plugin
+====================
+
+The Cargo package plugin can be used for Rust projects that are dependencies of
+other Rust packages. It is a companion plugin meant to be used with the
+:ref:`Rust plugin <craft_parts_rust_plugin>`. Use of this plugin makes rust
+builds in **all** parts happen offline.
```

Comment:
```suggestion
builds in *all* parts happen offline.
```

Reply:
Thanks; I see this now under [Bold vs. Italics](https://docs.ubuntu.com/styleguide/en#interacting-with-ui-elements)